README.md

```diff
@@ -165,17 +173,24 @@ Make sure to replace the parameters accordingly.
 10. Trigger the DAG with the following:
 
 ```bash
-airflow trigger_dag -c  "{\"params\": { \"resource_id\": \"my-res-id-123\", \
-\"schema_fields_array\": [ \"field1\", \"field2\" ], \
-\"csv_input\": \"/path/to/my.csv\", \
-\"json_output\": \"/path/to/my.json\" } }" \
-ckan_api_load_single_step
+airflow trigger_dag ckan_api_load_single_step \
+ --conf='{ "resource_id": "my-res-id-123", "schema_fields_array": [ "field1", "field2" ], "csv_input": "/path/to.csv", "json_output": "/path/to.json" }'
 ```
 
 Do not forget to properly replace the parameters with your data and properly escape the special characters.
 Alternatively, you can just run the DAG with the `airflow run` command.
 
-### Example 3: Local file to CKAN DataStore using Postgres
+
+`api_ckan_load_single_node` also works for remote CKAN instances. Just set up your Airflow `CKAN_SITE_URL` variable accordingly.
+
+
+#### Multiple-node DAG
+
+`ckan_api_load_multiple_steps` does the same steps of `api_ckan_load_single_node`, but it uses multiple nodes (tasks). You can repeat the steps of the previous section and run `ckan_api_load_multiple_steps`.
+
+
```

Comment:
Added `timedelta(hours=4)` to get the things working this way:
` 'start_date': datetime.now() - timedelta(hours=4),`
So, the DAG is triggered properly now, but get the following error on runtime:
```
airflow trigger_dag ckan_api_load_multiple_steps  --conf='{ "resource_id": "be9f63d3-40c1-4261-acb1-232bf675da64", "schema_fields_array": [ 'id', 'full_text', 'FID' ], "csv_input": "/home/michael/airflow/example.csv", "json_output": "/home/michael/airflow/my.json" }'
[2020-06-25 12:12:44,673] {__init__.py:51} INFO - Using executor SequentialExecutor
[2020-06-25 12:12:44,673] {dagbag.py:396} INFO - Filling up the DagBag from /home/michael/PycharmProjects/aircan_project/aircan/aircan/dags/api_ckan_load_multiple_nodes.py
Traceback (most recent call last):
  File "/home/michael/.local/bin/airflow", line 37, in <module>
    args.func(args)
  File "/home/michael/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 75, in wrapper
    return f(*args, **kwargs)
  File "/home/michael/.local/lib/python3.6/site-packages/airflow/bin/cli.py", line 237, in trigger_dag
    execution_date=args.exec_date)
  File "/home/michael/.local/lib/python3.6/site-packages/airflow/api/client/local_client.py", line 34, in trigger_dag
    execution_date=execution_date)
  File "/home/michael/.local/lib/python3.6/site-packages/airflow/api/common/experimental/trigger_dag.py", line 141, in trigger_dag
    replace_microseconds=replace_microseconds,
  File "/home/michael/.local/lib/python3.6/site-packages/airflow/api/common/experimental/trigger_dag.py", line 86, in _trigger_dag
    run_conf = json.loads(conf)
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 83 (char 82)

```

Reply:
Look at your `"schema_fields_array"`, they must contain fields in double quotes: ex: `"schema_fields_array": [ "FID", "Mkt-RF", "SMB", "HML", "RF" ]`